donkeycar/parts/pytorch/torch_data.py

```diff
@@ -41,7 +41,7 @@ def get_default_transform(for_video=False, for_inference=False):
     return transform
 
 
-class TorchTubDataset(IterableDataset):
+class FastaAiTorchTubDataset(IterableDataset):
```

Comment:
Looks like this class has now been defined twice, pls check comment above.

Reply:
This has been reverted.